frontend/src/metabase/dashboard/components/DashCard/DashCard.tsx

```diff
@@ -116,6 +116,9 @@ function DashCard({
   const [isPreviewingCard, setIsPreviewingCard] = useState(false);
   const cardRootRef = useRef<HTMLDivElement>(null);
 
+  const [isActionPanelSubmenuOpen, setIsActionPanelSubmenuOpen] =
```

Comment:
This is the state mentioned in the PR body, that I'd like to move to a new component that replaces `renderDashCardActions`

Reply:
I'll try to draft a stacked pr on top of this